app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.databinding
+
+import android.text.format.DateUtils
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import java.util.*
+
+/** Converts time in ms to readable relative time string. */
+@BindingAdapter("profile:date")
```

Comment:
Are we using this anywhere? If not, please remove.

Reply:
Removed file.